### PR TITLE
fix(application-name): adds default name for application is none is s…

### PR DESCRIPTION
**Pull request by invincibleJai**
- adds default name for an application if none is selected

Tracks: https://jira.coreos.com/browse/ODC-1646

### CODE REVIEW

File: frontend/packages/dev-console/src/components/import/image-search/ImageSearch.tsx [hunk 1]
```diff
@@ -63,6 +63,8 @@ const ImageSearch: React.FC = () => {
           setFieldValue('isi.ports', ports);
           setFieldValue('image.ports', ports);
           values.name === '' && setFieldValue('name', getSuggestedName(name));
+          values.application.name === '' &&
```
Comment (christianvogt) [inline frontend/packages/dev-console/src/components/import/image-search/ImageSearch.tsx]: In one spot it checks `!values.application.name` and here it does `values.application.name === ''`. 
The checks should be consistent.

Comment (invincibleJai) [reply frontend/packages/dev-console/src/components/import/image-search/ImageSearch.tsx]: will update it, I did based on above line `values.name === ''`

**Pull request merged**